storage/pkg/flagset/storagemetadata.go

```diff
@@ -73,7 +88,15 @@ func StorageMetadata(cfg *config.Config) []cli.Flag {
 	flags = append(flags, DriverLocalWithConfig(cfg)...)
 	flags = append(flags, DriverOwnCloudWithConfig(cfg)...)
 	flags = append(flags, DriverOCISWithConfig(cfg)...)
-
+	flags = append(flags,
```

Comment:
@IljaN we need to apend the `storage-root` to the end, otherwise the evaluation of the config flags for the drivers will overwrite the default that was written into `&cfg.Reva.Storages.Common.Root`.